mobly/controllers/attenuator_lib/telnet_scpi_client.py

```diff
@@ -72,6 +73,9 @@ def cmd(self, cmd_str, wait_ret=True):
     match_idx, match_val, ret_text = self._tn.expect(
         [_ascii_string("\S+" + self.rx_cmd_separator)], 1)
 
+    logging.debug('Telnet Command: %s', cmd_str)
```

Comment:
let's log these info in one line to be consistent with the other cmd logging patterns in Mobly, like `adb`